### PR TITLE
Updates README.md to be more compelling

### PR DESCRIPTION
**Pull request by Richard-Gist**
## Checklist:
- [ ] Is the linter reporting 0 errors?
- [ ] Did you comply with our [styleguide](https://github.com/wwt/Workflow/blob/main/STYLEGUIDE.md)?
- [ ] Is there [adequate test coverage](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#test-etiquette) for your new code?
- [ ] Does the CI pipeline pass?
- [ ] Is the [static analysis tool](https://sonarcloud.io/dashboard?id=wwt_Workflow) reporting 0 issues?
- [X] Did you [update the documentation](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#documentation)?
- [ ] Did you [update the sample app](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#sample-app)?
- [ ] Do we need to [increment the minor/major version](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#versioning)?
- [ ] Did you [change the public API](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#public-api)?
- [ ] Have you done everything you can to make sure that errors that can occur are compile-time errors, and if they have to be runtime do you have adequate tests and documentation around those runtime errors? [For more details](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#errors).

----

### If Applicable:
- [ ] Did you test when the first item is skipped?
- [ ] Did you test when the last item is skipped?
- [ ] Did you test when middle items are skipped?
- [ ] Did you test when incorrect data is passed forward?
- [ ] Did you test proceeding backwards?

----

### If Public API Has Changed:
- [ ] Did you deprecate (rather than remove) any old methods/variables/etc? [Our philosophy for deprecation](https://github.com/wwt/Workflow/blob/main/CONTRIBUTING.md#deprecation).
- [ ] Have you done the best that you can to make sure that the compiler guides people to changing to the new API? (Example: the renamed attribute)
- [ ] If necessary, have you tested the upgrade path for at least N-1 versions? For example, if data persists between v1 and v2 then that upgrade should be tested and as easy as we can make it.

### CODE REVIEW

**Comment by Richard-Gist** [comment]
EOD: Brought in Wiki files to source so that we can codify the changes.  We also improved our tone and voice to be better on brand (more to go).

**Comment by Richard-Gist** [comment]
EOD: Decided that the original direction of Getting-Started was more suited as a Tutorial and a more slimmed down, succinct version should be used instead.

File: wiki/Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+So you're interested in trying this out. Start by cloning the repo and checking out the 'WorkflowExample' scheme. This should give you a decent idea of how the library works conceptually.
```
Comment (Tyler-Keith-Thompson) [inline wiki/Getting-Started.md:1]: This only conceptually tells you how it works? Conceptually as a word really bothers me here. I keep rereading it and it's a little off-putting about everything that comes next.

File: wiki/Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+1) `MainStoryboardLoadable` inherits from [StoryboardLoadable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/StoryboardLoadable.html) which can only be applied to a `UIViewController` that is also a [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html).
+1) These view controllers are [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) because they implement the protocol through [UIWorkflowItem](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/UIWorkflowItem.html) AND are required to be [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) by [StoryboardLoadable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/StoryboardLoadable.html).
```
Comment (Tyler-Keith-Thompson) [inline wiki/Getting-Started.md:80]: They partially implement the protocol, but they don't really implement it. It does not implement init, for example.

File: wiki/Getting-Started.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        }
+    }
+}
+```
+### Let's discuss what's going on here.
+#### **Where is the type safety, I heard about?**
+The [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html) has compile-time type safety on the Input/Output types of the supplied [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html)s. This means that you will get a build error if the output of `FirstViewController` does not match the input type of `SecondViewController`.
+
+#### **What's going on with this `passedArgs`?**
+The `onFinish` closure for `launchInto(_::)` provides the last passed [AnyWorkflow.PassedArgs](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/AnyWorkflow/PassedArgs.html) in the work flow. For this Workflow, that could be the output of `FirstViewController` or `SecondViewController` depending on the email signature typed in `FirstViewController`. To extract the value, we unwrap the variable within the case of `.args()` as we expect this workflow to return some argument.
```
Comment (Tyler-Keith-Thompson) [inline wiki/Getting-Started.md:115]: `launchInto(_::)` or `launchInto(_:)`???

Blame Richard, he told me to review 😛

**Comment by Richard-Gist** [comment]
Just dropping a note here to run Jazzy again so that it has the updated README for its documentation.

**Comment by Richard-Gist** [comment]
Someone please create a Change Request to generate Jazzy documentation so we don't forget.  I don't want to do it now because it obscures the important changes we made.

EOD:  We rechecked our tone compared to what WWT expects in our branding, and revised Installation to match that voice. 
 We want to help people find the information needed to get started with our project, but we believe other resources will do a better job than this Github page.

File: wiki/Installation.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Swift Package Manager
```
Comment (wiemerm) [inline wiki/Installation.md:1]: We should have someone newer to SPM test that this makes sense.

File: wiki/Why-This-Library.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+### Why This Library?
+There are many reasons to be excited about Workflow:
+- Build flexibility into your applications so that you can easily make changes when you need to.
+- Compose new `Workflow`s to quickly create a new journey through your app.
```
Comment (wiemerm) [inline wiki/Why-This-Library.md:4]: Make Workflow a link (including the 's') as it is referring to a type.

File: wiki/Why-This-Library.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- Power deep linking experiences that can get your users right where you want them with minimal overhead.
+
+Read on to learn about our philosophy of development. We believe these ideals take us from "just another library" too a tool you won't want to do without.
+
+### Developer Experience over Problem Solving
+There have been many attempts by others to create libraries that decouple the concept of a flow that can move forward and backwards, but they all took very different approaches. What separates us, and makes this library worth using, is our developer-centric experience. We sat down with a group of developers and asked the question "if something existed today that did what you wanted, what would that thing look like?" then we spent a lot of effort making that happen.
+
+Additionally, this is a concerted effort from WWT. We do a lot of software development for large companies and this problem space is something we've regularly encountered. The concept of a `Workflow` has been toyed with by several projects. This library has been used to power some really great experiences already, and we intend to continue using it to do even bigger and better things in the future.
```
Comment (wiemerm) [inline wiki/Why-This-Library.md:14]: This Workflow is not referring to a type and should not be linked and/or coded.

File: wiki/Why-This-Library.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+### Flexibility over Assumptions
+Not everyone develops software the same way, and it would be a sad day for the industry if we thought we had all the answers. We believe strongly in a culture of innovation and we want consumers of our library to be able to make choices they feel are correct. `Workflow`s are not an architectural pattern, they're an abstraction you can choose to use. Much like the standard libraries that come with programming languages the idea here is to give you tools that can be really useful and let you choose the best way to use them.
```
Comment (wiemerm) [inline wiki/Why-This-Library.md:17]: This falls under one of the previous 2 Workflow comments. Decide which one and fix it.

**Review by wiemerm — CHANGES_REQUESTED** [review]
The commented out gif to be added either make a card or finish it in this. The rest to be fixed.

File: README.md [hunk 8]
```diff
@@ -3,42 +3,68 @@
 ![Quality Gate](https://img.shields.io/sonar/quality_gate/wwt_Workflow?server=https%3A%2F%2Fsonarcloud.io)
 ![Coverage](https://img.shields.io/sonar/coverage/wwt_Workflow?server=http%3A%2F%2Fsonarcloud.io)
 
-### Welcome to Workflow!
-Workflow is a library meant to handle complex user flows in iOS applications.
+# Welcome
```
Comment (wiemerm) [inline README.md:6]: Generate Jazzy documentation

**Review by wiemerm — CHANGES_REQUESTED** [review]
Do the thing it says.

File: STYLEGUIDE.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ Note that brevity is not a primary goal. Code should be made more concise only i
 
 ## Guiding Tenets
 
-* The [official swift API design guidelines](https://swift.org/documentation/api-design-guidelines/) are all unilaterally accepted for any public API and generally have good information for private or internal APIs. If you are not making a public API we do not require the same level of documentation, but the naming conventions and general design guidelines are still great and should be adhered to.
+* The [official Swift API design guidelines](https://swift.org/documentation/api-design-guidelines/) are all unilaterally accepted for any public API and generally have good information for private or internal APIs. If you are not making a public API we do not require the same level of documentation, but the naming conventions and general design guidelines are still great and should be adhered to.
```
[the remark this answers is not in the document]
Comment (Richard-Gist) [reply STYLEGUIDE.md:15]: done

File: wiki/Getting-Started.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+#### **Why these initializers?**
+<details>
+
+[StoryboardLoadable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/StoryboardLoadable.html) helps guide XCode to give you compiler errors with the appropriate fix-its to generate `required init?(coder: NSCoder, with args: String)`. These initializers allow you to load from a storyboard while also having compile-time safety in your properties.  You will notice that both view controllers store the argument string on a `private let` property.
+</details>
+
+#### **What's this `shouldLoad()`?**
+<details>
+
+It is part of the [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience, but is still left implementable by you should you want to control when a [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) should load in the work flow.  It is called after `init` but before `viewDidLoad()`.
```
[the remark this answers is not in the document]
Comment (Richard-Gist) [reply wiki/Getting-Started.md:103]: I just noticed I could commit your suggestions, but instead I'm doing it myself....

File: wiki/Why-This-Library.md (hunk 7 above)
Comment (Richard-Gist) [reply wiki/Why-This-Library.md]: I decided concept, and removed formatting.

File: README.md [hunk 11]
```diff
@@ -3,42 +3,68 @@
...
 
+<!-- See it in action!
+<details>
+  <summary><b>Storyboard based application</b></summary>
+	[GIF]
+</details>
+<details>
+  <summary><b>Programmatic view based application</b></summary>
+	[GIF]
+</details> -->
```
[the remark this answers is not in the document]
Comment (Richard-Gist) [reply README.md]: Card made on the Kanban board, including the comment itself.  Removed these lines.

File: wiki/Installation.md (hunk 4 above)
Comment (Richard-Gist) [reply wiki/Installation.md:1]: Made a card to do this: https://github.com/wwt/Workflow/projects/1#card-61988366

**Review by wiemerm — APPROVED** [review]

**Pull request merged**